backend/src/utils/common.ts

```diff
@@ -1,7 +1,33 @@
 import { FastifyBaseLogger, FastifyRequest } from "fastify";
+import SupportedNetworks from "../../config.json" assert { type: "json" };
+import { Database } from "sqlite3";
 
-export function printRequest(methodName: string, request: FastifyRequest, log: FastifyBaseLogger) {
-  log.info(methodName, "called: ");
+export function printRequest(request: FastifyRequest, log: FastifyBaseLogger) {
   log.info(request.query, "query passed: ");
```

Comment:
Why is this replaced? I had included the function name as well.

Reply:
Sorry didnt see that as you said you only took off wait for deposit transaction. will add it